includes/core/server/class-request.php

```diff
@@ -379,7 +409,7 @@ protected function set_request_route_id_parameter( $id ) {
 	 * @param string $key   The name of the parameter.
 	 * @param mixed  $value And the value to set.
 	 */
-	final protected function set_param( string $key, $value ) {
+	final public function set_param( string $key, $value ) {
```

Comment:
This looks wrong. Making this method public makes all other protections effectively pointless.

Reply:
Strange. I thought that I removed this line. I will undo this.

Reply:
Addessed here: https://github.com/Automattic/woocommerce-payments/pull/7314/commits/6a78ee9776f280116ebe4519e649cb2d692f0eeb